test/system/adopter_application_edit_test.rb

```diff
@@ -11,10 +11,12 @@ class AdopterApplicationEditTest < ApplicationSystemTestCase
   test "Clicking the information icon makes the information box appear and disappear" do
     assert_selector "h1", text: "Ben Jo's application for Adopted"
     assert_selector "p.explanation", count: 0
-    find("img.ms-2").click
-    assert_selector "p.explanation", count: 1
-    find("img.ms-2").click
-    assert_selector "p.explanation", count: 0
+
+    find("img.ms-2").click do |image|
```

Comment:
Ah nicely done! This was the one flaky test of the suite. I am glad you found a way to fix that.